### PR TITLE
Update versioning

### PR DESCRIPTION
**Pull request by AleksSenkou**

### CODE REVIEW

File: roles/ruby/tasks/rbenv.yml [hunk 1]
```diff
@@ -1,25 +1,27 @@
...
+
 - name: Installs rbenv
   git: repo=git://github.com/sstephenson/rbenv.git dest=~{{ deploy_user }}/.rbenv
 
 - name: Ensure fragments bash dir exists
   file: dest=~{{ deploy_user }}/.bash.d/ state=directory
 
 - name: Installs rbenv bash file
-  copy: src=50_rbenv.bash dest=~{{ deploy_user }}/.bash.d/50_rbenv.bash mode=700 owner={{ deploy_user }}
+  copy: src={{ rbenv_dependencies_file_name }} dest=~{{ deploy_user }}/.bash.d/{{ rbenv_dependencies_file_name }} mode=700 owner={{ deploy_user }}
```
Comment (roman-dubrovsky) [inline roles/ruby/tasks/rbenv.yml:11]: @AleksSenkou why did you add variable for this file name? Do you really need it?

File: roles/common/tasks/nodejs.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+- name: Fetch latest Nodejs 7.x installing script
+  get_url:
+    url: https://deb.nodesource.com/setup_7.x
+    dest: /root/nodesource
+
+- name: Add Nodejs support
+  command: bash /root/nodesource
+
+- name: Install Nodejs
+  apt: name=nodejs state=latest
```
Comment (roman-dubrovsky) [inline roles/common/tasks/nodejs.yml:11]: @AleksSenkou if you want to fix installing nodejs, you should:

1) Install nodejs using nodeenv
2) Install npm
3) Install yarn

**Review by roman-dubrovsky — CHANGES_REQUESTED** [review]

File: roles/ruby/tasks/rbenv.yml (hunk 1 above)
Comment (AleksSenkou) [reply roles/ruby/tasks/rbenv.yml:11]: @roman-dubrovsky yeap, it used in several places

Comment (roman-dubrovsky) [reply roles/ruby/tasks/rbenv.yml:11]: @AleksSenkou ok, and why did you create variable only for this file?

**Review by roman-dubrovsky — APPROVED** [review]
